app/src/main/res/values/strings.xml

```diff
@@ -418,7 +418,7 @@
   <string name="articles">articles</string>
 
   <string name="no_uploads">Welcome to Commons!\n
-Upload your first media by touching the camera or gallery icon above.</string>
+Upload your first media by tapping on the button below.</string>
```

Comment:
Minor Detail: just because there may be more buttons "below" as the app evolves or depending on the user device, we might want to be specific and say "tapping on the add button below". I don't know if that's better though so it's up to you if you want to change it.